src/components/icons/IconWrapper.tsx

```diff
@@ -47,3 +47,21 @@ const StyledSpan = styled.span`
   -moz-osx-font-smoothing: grayscale;
   text-rendering: optimizeLegibility;
 `;
+IconMaterialWrapper.displayName = 'IconMaterialWrapper';
+
+export const IconCustomWrapper = styled(IconBase)`
+  font-family: 'Echoes Icons';
+  font-size: 1.2em;
+  font-weight: normal;
+  font-style: normal;
+  line-height: 1;
+  letter-spacing: normal;
+  text-transform: none;
+  white-space: nowrap;
+  word-wrap: normal;
+  direction: ltr;
```

Comment:
do we really need this one (`direction`) ?

Reply:
Probably not, this whole style comes from the class provided by the material UI lib, there are probably many things we can clean up.